planet/scripts/v1.py

```diff
@@ -415,10 +415,21 @@ def get_mosaic_list_for_feed(feed_id):
     for type_ in ['target', 'source']:
         feed_image_conf = feed_info.get(type_)
 
-        if feed_image_conf['type'] != 'mosaic':
-            msg_format = 'The {} for this feed is not a mosaic type.'
-            click.ClickException(msg_format.format(type_))
-            continue
```

Comment:
This code has the looks of a accidentally-committed partial solution?

Reply:
Here's the commit https://github.com/planetlabs/planet-client-python/commit/ef726a53e32c12a4c8ad8885063d764e0c3044c5.